tfhe/src/integer/public_key/tests.rs

```diff
@@ -7,12 +7,14 @@ use crate::shortint::ClassicPBSParameters;
 use crate::integer::keycache::KEY_CACHE;
 
 create_parametrized_test!(big_radix_encrypt_decrypt_128_bits {
+    #[cfg(not(feature = "__coverage"))]
```

Comment:
I think we can better use the cfg tarpaulin ?

https://github.com/xd009642/tarpaulin?tab=readme-ov-file#ignoring-code-in-files

Maybe in a follow-up PR to avoid having any more changes on this PR